tests/settings.py

```diff
@@ -117,7 +117,13 @@
 #:   {"batch": {"nodegroup": ["compute"]}, "interactive": {"nodegroup": ["interactive"]}}
 #: The keys in the scheduling_class dictionary must correspond to the lower
 #: case name of the process scheduling class.
-FLOW_KUBERNETES_AFFINITY = os.environ.get("RESOLWE_KUBERNETES_AFFINITY", None)
+FLOW_KUBERNETES_AFFINITY = None
```

Comment:
Why is this changed in this commit?

Reply:
You are right, it deserves a commit of its own. 

I made a stupid mistake to read a JSON encoded variable from environment directly, which would not work. Since we do not really need this in Resolwe I decided to add a default value in the settings file.